web/client/src/library/components/plan/PlanOptions.tsx

```diff
@@ -142,8 +138,16 @@ export default function PlanOptions({
                         info="Ensure that new snapshots have no data gaps when
               comparing to existing snapshots for matching
               models in the target environment"
-                        enabled={Boolean(no_gaps)}
-                        disabled={isInitialPlanRun}
+                        enabled={
```

Comment:
Can InputToggle be enabled and disabled at the same time?

Reply:
`enabled` is referring to Toggle State but `disabled` is referring to input state, so `enabled` Toggle can be `disabled`. It is poor naming, i will rename it to something like `checked` in separate PR